packages/algolia-search/lib/configuration.ts

```diff
@@ -0,0 +1,3 @@
+export const applicationId = import.meta.graphCommerce.algoliaApplicationId ?? ''
+export const searchOnlyApiKey = import.meta.graphCommerce.algoliaSearchOnlyApiKey ?? ''
+export const searchIndex = import.meta.graphCommerce.algoliaSearchIndex ?? ''
```

Comment:
If we make the fields required:

```suggestion
export const applicationId = import.meta.graphCommerce.algoliaApplicationId
export const searchOnlyApiKey = import.meta.graphCommerce.algoliaSearchOnlyApiKey
export const searchIndex = import.meta.graphCommerce.algoliaSearchIndex
```

Reply:
Also, this doesn't take the storefront override in consideration

Reply:
Might it be an idea to create a useAlgoliaConfiguration() hook which also allows us to use the useStoreconfig hook so we can override the configuration for a specific storefront?

Reply:
De plugin supports storefront specific search indexes and configurations, but the application id is app based. Do you think we need to make this storefront specific as well?